pages/signup.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login</title>
+    <link rel="stylesheet" href="https://component-site-fitui.netlify.app/components/components.css">
+    <link rel="stylesheet" href="../css/header.css">
+    <link rel="stylesheet" href="../css/login.css">
+    <link rel="stylesheet" href="../css/signup.css">
+    <link rel="stylesheet" href="../style.css">
+    <script src="https://code.iconify.design/2/2.1.2/iconify.min.js"></script>
+</head>
+
+<body>
+    <div class="grid-layout">
+        <header class="header">
+            <div class="navbar">
+                <h1 class="navbar-title">Fit Store</h1>
+                <div class="navbar-search">
+                    <input type="text" placeholder="search..." class="navbar-input"><span
+                        class="iconify navbar-search-icon" data-icon="bx:bx-search" data-inline="false"></span>
+                </div>
+                <div class="navbar-icons">
+                    <a href="#" class="btn btn-link navbar-login">login</a>
+                    <a class="navbar-wishlist" href="#"><span class="iconify navbar-wishlist-icon"
+                            data-icon="mdi:cards-heart-outline" data-inline="false"></span></a>
+                    <a class="btn btn-link navbar-cart" href="#"><span class="iconify navbar-cart-icon"
+                            data-icon="zmdi:shopping-cart" data-inline="false"></span></a>
+                </div>
+            </div>
+        </header>
+        <div class="form">
+            <h2 class="title-login">Signup</h3>
```

Comment:
opening is a h2 tag but closing is a h3 tag.

Reply:
thanks for point out the mistake